man/man8/zfs.8

```diff
@@ -1056,6 +1111,16 @@ This property is not used on Linux.
 .sp
 .ne 2
 .na
+\fB\fBpbkdf2iters\fR=\fIiterations\fR
+.ad
+.sp .6
+.RS 4n
+Controls the number of PBKDF2 iterations that a \fBpassphrase\fR encryption key should be run through when processing it into an encryption key. This property is only defined when encryption is enabled and a \fBpassphrase\fR key is selected. The goal of PBKDF2 is to significantly increase the computational difficulty needed to brute force a user's passphrase. This is accomplished by forcing the attacker to run each passphrase through a computationally expensive hashing function many times before they arrive at the resulting key. A user who actually knows the passphrase will only have to pay this cost once. As CPUs become better at processing, this number should be raised to ensure that a brute force attack is still not possible. The current default (if not otherwise specified) is \fB100000\fR This may be changed with \fBzfs key -c\fR.
```

Comment:
> Maybe this should say the property can only be set then?

The property can either be undefined (if the dataset is not encrypted), the default value (when not specified by the user) or user defined. I'm not sure if your phrasing conveys the entire meaning.

> That's redundant, it's what "default" means.

Will fix.